Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
"Friends" implies plural but it's really just `after(:suite)`, right?  Might as well say:

> Correctly run `before(:suite)` and `after(:suite)` hooks in the context of an example group...